saleor/graphql/checkout/tests/mutations/test_checkout_complete_with_payment.py

```diff
@@ -1110,12 +1110,16 @@ def test_checkout_complete_with_voucher_apply_once_per_order(
 
     code.refresh_from_db()
     assert code.used == voucher_used_count + 1
-    order_discount = order.discounts.filter(type=DiscountType.VOUCHER).first()
-    assert order_discount
+    order_line_discount = OrderLineDiscount.objects.get()
+    assert order_line_discount
     assert (
-        order_discount.amount_value
+        order_line_discount.amount_value
         == (order.undiscounted_total - order.total).gross.amount
     )
+    assert order_line_discount.type == DiscountType.VOUCHER
+    assert order_line_discount.voucher == voucher_percentage
+    assert order_line_discount.voucher_code == code.code
```

Comment:
can you test discount value type in all the tests?
`assert order_line_discount.value_type == DiscountValueType.PERCENTAGE`